README.rst

```diff
@@ -1,5 +1,5 @@
-.. image:: https://secure.travis-ci.org/ActiveState/appdirs.png
-    :target: https://travis-ci.org/ActiveState/appdirs
+.. image:: https://github.com/jdufresne/appdirs/workflows/Test/badge.svg
+   :target: https://github.com/jdufresne/appdirs/actions?query=workflow%3ATest
```

Comment:
Remember to change these back to upstream after merge.

Reply:
Thanks for catching that. Updated in latest revision.